src/clients/api/queries/useGetXvsBalanceOf.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery, QueryObserverOptions } from 'react-query';
+
+import getXvsBalanceOf, {
+  IGetXvsBalanceOfInput,
+  GetXvsBalanceOfOutput,
+} from 'clients/api/queries/getXvsBalanceOf';
+import FunctionKey from 'constants/functionKey';
+import { useTokenContract } from 'clients/contracts/hooks';
+
+type Options = QueryObserverOptions<
+  GetXvsBalanceOfOutput,
+  Error,
+  GetXvsBalanceOfOutput,
+  GetXvsBalanceOfOutput,
+  FunctionKey.GET_XVS_BALANCE_OF
+>;
+
+const useGetXvsBalanceOf = (
+  { accountAddress }: Omit<IGetXvsBalanceOfInput, 'xvsTokenContract'>,
+  options?: Options,
+) => {
+  const xvsTokenContract = useTokenContract('xvs');
+
+  return useQuery(
+    FunctionKey.GET_XVS_BALANCE_OF,
+    () => getXvsBalanceOf({ xvsTokenContract, accountAddress }),
+    options,
+  );
+};
+
+export default useGetXvsBalanceOf;
```

Comment:
I'd suggest creating a `useGetTokenBalance` instead, similarly to `useGetVTokenBalance`, that can then be used to get the balances for XVS and VRT (and any other token we support that's not a Venus token).